README.md

```diff
@@ -102,3 +102,52 @@ export const writeFileFromText = (text, pathname) => {
   return writeFileSync(join(projectPath, pathname), text, 'utf-8')
 }
 ```
+
+```js
+// converter.test.js
+import { expect } from 'chai'
+import { convertLineToContext, convertLinesToContexts } from './converter';
+import { getTextFromFile } from './file'
+
+describe('test converter', () => {
+  it('Should convert readme to code', () => {
+    const README = getTextFromFile(`./README.md`)
+    const lines = README.split('\n')
+
+    expect(convertLinesToContexts(lines)).is.an('array')
+  })
+  
+  it('Should convert line to context if line has c2e:', () => {
```

Comment:
This case should be `should convert line to context if this line matches code-to-essay template`.